src/index.ejs

```diff
@@ -14,6 +14,9 @@
     <title>(karrot loading...)</title>
     <link rel="shortcut icon" href="statics/carrot_logo.ico">
     <link rel="icon" type="image/png" href="statics/carrot_logo.png" sizes="32x32">
+    <% if (CORDOVA) { %>
```

Comment:
Can this also get loaded via webpack import?

Reply:
It's a buildtime configuration option, not runtime, the built files do not contain this. It's defined using the `WebpackDefine` plugin.